kahuna/public/js/search/query.js

```diff
@@ -23,7 +23,7 @@ query.controller('SearchQueryCtrl',
                  ['$scope', '$state', '$stateParams', 'onValChange', 'mediaApi', 'track',
                  function($scope, $state, $stateParams, onValChange , mediaApi, track) {
 
-    var ctrl = this;
```

Comment:
I don't think it would as wee add properties to it. e.g. [`ordering`](https://github.com/guardian/grid/blob/decode-uri/kahuna/public/js/search/query.js#L28-L30)